recipes/bullet3/all/bullet_extra.cmake

```diff
@@ -0,0 +1,10 @@
+set(BULLET_INCLUDE_DIR "${Bullet_INCLUDE_DIRS}")
```

Comment:
I understand that these lines are provided to allow the consumer to locate the bullet library. right?
All that should be provided automatically by Conan using a generator.

Reply:
Correct, because the `cmake_find_package` file must be named `FindBullet.cmake`, conan created `Bullet_INCLUDE_DIRS` cmake variables.
Since cmake is case sensitive, I add `BULLET_INCLUDE_DIRS` as a copy.